<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="numberLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="responseLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите 32-хразрядное число</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите k-ый бит числа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ответ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setUnsetButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить. Снять</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setZeroButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Обнулить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Поменять местами биты</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
